.github/workflows/pr.yaml

```diff
@@ -0,0 +1,80 @@
+on: pull_request
+name: Review
+jobs:
+    test-composer-install:
+        name: Validate composer
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@master
+            -   name: Setup PHP, with composer and extensions
+                uses: shivammathur/setup-php@master
+                with:
+                    php-version: 7.3
+                    extension-csv: ctype, iconv, json
+                    coverage: none
+            -   name: Validate composer files
+                run: |
+                    composer validate composer.json
+            -   name: Composer install with exported .env variables
+                run: |
+                    set -a && source .env && set +a
```

Comment:
Why is this needed? Should composer/symfony not do this for you?

Reply:
Not when running composer commands

Reply:
Not sure I understand this... I don't do this when building docker images with Symfony applications. What is in `.env` thats need during `composer install` here not normally required. It might be me that is missing something here...

Reply:
Composer install runs cache:clear which invokes symfony. So if environment variables are not set, this will provoke an exception

Reply:
I will try commenting out the line, to see if it is needed

Reply:
Fixed